### DISCUSSION TITLE
Merge cells in DataGrid header

### DISCUSSION THREAD

**Discussion by To1iK** [microsoft/fluentui-blazor#1883] [Q&A] [answered]
Is it possible to merge cells in FluentDataGrid header like bellow 
![image](https://github.com/microsoft/fluentui-blazor/assets/30840941/c57b9989-cd64-43ab-9185-8dde0dc28a4b)

**Comment by vnbaaij** [comment, marked as answer]
No, that is not possible. The Datagrid web components have no notion of a span or something like that.
